q2_covid/common.py

```diff
@@ -45,3 +48,73 @@ def __init__(self, inclusion: pd.Series, metadata: qiime2.Metadata,
 
 
 Selection = SemanticType('Selection', variant_of=FeatureData.field['type'])
+
+# modified from DNAFastaFormat in q2_types to allow lowercase characters
+# https://github.com/qiime2/q2-types/blob/058ee0e40e38edaa02b1aad034df37456aeb4ddf/q2_types/feature_data/_format.py#L146
+class GISAIDDNAFASTAFormat(model.TextFileFormat):
+    def _validate_lines(self, max_lines):
```

Comment:
One idea that came to my mind was to factor out the `DNAFastaFormat` validation code into an exported util, and then importing that here. I imagine that util could accept the alphabet as input. Small bit of effort required, but might cut down on any issues with maintaining two separate formats that are almost 100% identical, and have pretty involved business logic.

Reply:
Alternatively, instead of exporting a util, we could tweak `DNAFastaFormat` to be a base class, kinda like the SLPSSEDF formats, and then `GISAIDDNAFASTAFormat` could just be a subclass.

Neither of these suggestions are high priority (read: don't need to be addressed now), but should at least be recorded as an issue or two somewhere.

Reply:
@thermokarst, does [this issue](https://github.com/caporaso-lab/q2-covid-19/issues/7) cover this?

Reply:
LGTM, thanks.